release_notes/ocp-4-11-release-notes.adoc

```diff
@@ -1522,9 +1506,19 @@ sourceStrategy:
 
 * Previously, the registry used exact match when applying `ImageContentSourcePolicy`(ICSP). With this update, ICSP is applied for subrepositories and mirrors now work as expected. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2014240[*BZ#2014240*])
 
-[discrete]
-[id="ocp-4-11-image-streams-bug-fixes"]
-==== Image Streams
+* Previously, the Image Registry Operator did not work with AWS S3 on {rh-openstack}. With this update, the Image Registry Operator trusts AWS S3 on all platforms. (link:https://bugzilla.redhat.com/show_bug.cgi?id=200761[*BZ#2007611*])
+
+* Previously, {product-title} image registry did not work with Ceph Radosgw. With this update, the image registry works with Ceph Radosgw. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1976782[*BZ#1976782*])
+
+* Previously, the Image Registry Operator interpreted `429` error messages from upstream registries as if the data was not available. The Operator returned a `404 Not Found instead` message instead of `429 Too Many Requests`. With this update, the proper `429 Too Many Requests` message is returned so that administrators know to retry the request. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1923536[*BZ#1923536*])
+
+* Previously, the Image Registry Operator could not be used to configure CloudFront. With this update, the Image Registry Operator can configure CloudFront. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2065224[*BZ#2065224*])
+
+* Previously, the Image Registry Operator push images when KMS encryption was enabled. With this update, images can be pushed and pulled with KMS encryption enabled. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2048214[*BZ#2048214*])
```

Comment:
From this line, I'm not certain what the change is between the previous behaviour and the updated behaviour. Was it that previously the IMO could not push images when KMS was enabled? If so, you could update this line to reflect that.